File: ImobDLApi/DTOs/Filtros.cs
using ImobDLApi.models;

namespace ImobDLApi.DTOs
{
    public class Filtros
    {
        public List<Cidade> Cidades { get; set; }
        public List<Tipo> Tipos { get; set; }
        public decimal ValorMinV { get; set; }
        public decimal ValorMaxV { get; set; }
        public decimal ValorMinA { get; set; }
        public decimal ValorMaxA { get; set; }
    }
}

File: client-app/src/app/DTOs/filtros.ts
import {Cidade } from "../models/cidade";
import { Tipo } from "../models/tipo";

export default interface Filtros{
    cidades: Cidade[],
    tipos: Tipo[],
    valorMaxA: number,
    valorMinA: number,
    valorMaxV: number,
    valorMinV: number
}